questions/bind-methods-handlers-in-jsx-callbacks.md

```diff
@@ -0,0 +1,55 @@
+### How do you ensure methods used in callback props have the correct `this` context?
```

Comment:
I made many edits to this... let me know if the language / edits are satisfactory and I can apply changes to the rest

Reply:
I'm not sure how an interviewer would usually phrase this. I'm fine with the changes, iff they are more approachable from an interview standpoint.

Reply:
I'm currently AFC, so I can't modify this easily. It might sound better though to ask "How do you make sure methods have the correct context in React classes?"

Reply:
@skatcat31 yes you're right that's better